src/Components/Server/src/CircuitOptions.cs

```diff
@@ -46,5 +45,24 @@ public class CircuitOptions
         /// Defaults to <c>3 minutes</c>.
         /// </value>
         public TimeSpan DisconnectedCircuitRetentionPeriod { get; set; } = TimeSpan.FromMinutes(3);
+
+        /// <summary>
+        /// Gets or sets a value that determines whether or not to send detailed exception messages from .NET interop method invocation
+        /// exceptions to JavaScript.
+        /// </summary>
+        /// <remarks>
+        /// This value should only be turned on in development scenarios as turning it on in production might result in the leak of
+        /// sensitive information to untrusted parties.
+        /// </remarks>
+        /// <value>Defaults to <c>false</c>.</value>
```

Comment:
Can/should we default this based on `IsDevelopment`?

Reply:
Will be based on WebHostDefaults.DetailedErrorsKey